backend/README.md

```diff
@@ -33,3 +33,16 @@ All errors from backend should have in body:
   }
 }
 ```
+* Foce Update:
+	- error have to include a message to show in the alert
+	- error have to include an URL for the app redirect
+
+error response will be in the following format:
+```
+{
+  "errors": {
```

Comment:
The force update proposed by @Busta117 is ok, and it's being used now in all projects.

Let's continue the general error handling strategy on #18